drools-wb/base/Dockerfile

```diff
@@ -24,14 +23,19 @@ ENV JAVA_OPTS -XX:MaxPermSize=256m -Xms256m -Xmx512m
 USER jboss
 
 ####### DROOLS-WB ############
-RUN curl -o /opt/jboss/$DROOLS_CONTEXT_PATH.war $KIE_REPOSITORY/org/kie/kie-drools-wb-distribution-wars/$DROOLS_VERSION/kie-drools-wb-distribution-wars-$DROOLS_VERSION-$DROOLS_CLASSIFIER.war && \ 
-unzip -q /opt/jboss/$DROOLS_CONTEXT_PATH.war -d /opt/jboss/wildfly/standalone/deployments/$DROOLS_CONTEXT_PATH.war &&  \ 
-touch /opt/jboss/wildfly/standalone/deployments/$DROOLS_CONTEXT_PATH.war.dodeploy &&  \ 
-rm -rf /opt/jboss/$DROOLS_CONTEXT_PATH.war
+RUN curl -o $HOME/$KIE_CONTEXT_PATH.war $KIE_REPOSITORY/org/kie/kie-drools-wb-distribution-wars/$KIE_VERSION/kie-drools-wb-distribution-wars-$KIE_VERSION-$KIE_CLASSIFIER.war && \
+unzip -q $HOME/$KIE_CONTEXT_PATH.war -d $JBOSS_HOME/standalone/deployments/$KIE_CONTEXT_PATH.war &&  \
+touch $JBOSS_HOME/standalone/deployments/$KIE_CONTEXT_PATH.war.dodeploy &&  \
+rm -rf $HOME/$KIE_CONTEXT_PATH.war
+
+####### MVN REPO ############
+# https://bugzilla.redhat.com/show_bug.cgi?id=1263738
+RUN mkdir -p $HOME/.m2/repository/org/guvnor/guvnor-asset-mgmt-project/$KIE_VERSION && \
+curl -o $HOME/.m2/repository/org/guvnor/guvnor-asset-mgmt-project/$KIE_VERSION/guvnor-asset-mgmt-project-$KIE_VERSION.jar $KIE_REPOSITORY/org/guvnor/guvnor-asset-mgmt-project/$KIE_VERSION/guvnor-asset-mgmt-project-$KIE_VERSION.jar
 
 ####### EXPOSE INTERNAL JBPM GIT PORT ############
 EXPOSE 8001
 
 ####### RUNNING DROOLS-WB ############
-WORKDIR /opt/jboss/wildfly/bin/
-CMD ["./standalone.sh", "-b", "0.0.0.0", "--server-config=standalone-full.xml"]
\ No newline at end of file
+WORKDIR $JBOSS_HOME/bin/
+CMD ["./standalone.sh", "-b", "$JBOSS_BIND_ADDRESS", "-Dorg.kie.demo=$KIE_DEMO", "-Dorg.kie.example=$KIE_DEMO", "-c", "standalone-full.xml"]
```

Comment:
This will mot probably don't work, sine env variables are not available as part of `CMD` command executed as array.